cmd/lncli/cmd_import_mission_control.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/urfave/cli"
+)
+
+const argsStr = "[source node] [dest node] [unix ts seconds] [amount in msat]"
+
+var importMissionControlCommand = cli.Command{
+	Name:      "importmc",
+	Category:  "Payments",
+	Usage:     "Import a result to the internal mission control state.",
+	ArgsUsage: fmt.Sprintf("importmc %v", argsStr),
+	Action:    actionDecorator(importMissionControl),
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "failure",
+			Usage: "whether the routing history entry was a failure",
+		},
+	},
+}
+
+func importMissionControl(ctx *cli.Context) error {
```

Comment:
This is very nice, maybe could also add csv import? (Otherwise MC may change if node is busy).

Reply:
What do you by MC might change?

I was picturing the main use case for bulk-import to be the rpc? Since you can pass the request from `QueryMissionControl` directly in, adding a csv is an extra step. Could add an option to feed the json output in like we do for `SendToRoute` if we want to do bulk import via cli?